### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by luebken**
### Description of your changes

Updated readme to reflect recent product changes and guide users to build a custom configuration package.

Fixes #

I have:

- [ ] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,336 +1,243 @@
...
-  ```console
-  curl -sL https://cli.upbound.io | sh
+## Quickstart
 
-  ```
+### Pre-Requisites
 
-## Platform Ops/SRE: Run your own internal cloud platform
+Before we can install in the reference platform we need to install the `up` CLI
+and UXP. To install `up` run this install script:
```
Comment (ytsarev) [inline README.md:65]: It's not clear to the reader what the UXP is.

**Review by ytsarev — CHANGES_REQUESTED** [review]
I like the overall revamp, but found a couple of issues.

File: README.md [hunk 2]
```diff
@@ -1,336 +1,243 @@
...
-we can install this reference platform as a Crossplane package.
+Now you can install this reference platform. It's packaged as a [Crossplane
+configuration package](https://crossplane.io/docs/v1.9/concepts/packages.html)
+so there is a single command to install this package: 
 
 ```console
-# Check the latest version available in https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/
-kubectl apply -f examples/configuration.yaml
-kubectl get pkg
+kubectl crossplane install configuration xpkg.upbound.io/upbound/platform-ref-aws:v0.4.0
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md]: Fixed with https://github.com/upbound/platform-ref-aws/pull/79/commits/1cfb972a540bf91cc5de3ade1d156a98aaf129f6

File: README.md [hunk 3]
```diff
@@ -1,336 +1,243 @@
...
-and publishing to [Universal Marketplace](https://marketplace.upbound.io/).
-It enables you to build, run and operate your own internal cloud platform and
-offer a self-service API to your internal teams. It provides platform APIs to provision
-fully configured EKS clusters, with secure networking, and stateful cloud
-services (RDS) designed to securely connect to the nodes in each EKS cluster --
-all composed using cloud service primitives from the [Official Upbound AWS
+# AWS Reference Platform
+
+This repository contains a reference AWS Platform Configuration for
+[Crossplane](https://crossplane.io/). It's a great starting point for building
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md:4]: I intentionally used Crossplane here to pick the up the reader and let them make the connection to crosslplane. UXP is used throughout the rest of the guide.

File: README.md [hunk 4]
```diff
@@ -1,336 +1,243 @@
...
+open https://marketplace.upbound.io/configurations/${ORG}/${PLATFORM}:v0.0.1
 ```
 
+## Using your custom platform
+
+Now if you want to use it you can follow the steps from above. The only difference is that you need to specify a package-pull-secret as the package is currently private:
+
 ```console
-kubectl apply -f configuration.yaml
+kubectl crossplane install configuration xpkg.upbound.io/${ORG}/${PLATFORM}::v0.0.1 --package-pull-secrets=personal-pull-secret
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md]: fixed with https://github.com/upbound/platform-ref-aws/pull/79/commits/1cfb972a540bf91cc5de3ade1d156a98aaf129f6

File: README.md [hunk 5]
```diff
@@ -1,336 +1,243 @@
...
-Verify status:
+# Create a K8s secret with the AWS creds:
+kubectl create secret generic aws-creds -n upbound-system --from-file=creds=./creds.conf
 
-```console
-kubectl get claim
-kubectl get composite
-kubectl get managed
+# Configure the AWS Provider to use the secret:
+kubectl apply -f examples/aws-default-provider.yaml
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md:131]: thx. great catch. fixed with https://github.com/upbound/platform-ref-aws/pull/79/commits/18554f418edde2498588f0984fde31cb70c56f30

File: README.md (hunk 1 above)
Comment (luebken) [reply README.md]: Addressed with https://github.com/upbound/platform-ref-aws/pull/79/commits/2173f8f7e649d251f62ad43dc57eeca07cdf199e

File: README.md [hunk 6]
```diff
@@ -1,336 +1,248 @@
...
-
-Learn more about `Composition` in the [Crossplane
+## Overview
+
+This reference platform defines a custom API for creating an EKS cluster
+([XCluster](package/cluster/definition.yaml)) which includes the actual EKS
+cluster, a network fabric and Prometheus and other cluster services
+([XServices](package/cluster/composition.yaml)). Additionally it defines a
+custom API for provisioning Postgres Databases
+([XPostgreSQLInstance](package/database/postgres/definition.yaml)).
```
Comment (ytsarev) [inline README.md:23]:
```suggestion
([XPostgreSQLInstance](package/database/postgres/definition.yaml)).
```

File: README.md [hunk 7]
```diff
@@ -1,336 +1,248 @@
...
-Install UXP into the `upbound-system` namespace:
+Finally install UXP into the `upbound-system` namespace:
 
 ```console
-up login
 up uxp install
 ```
 
-Validate the install using the following command:
+You can validate the install by inspecting all installed components:
```
Comment (ytsarev) [inline README.md:92]:
```suggestion
You can validate the install by inspecting all installed components:
```

File: README.md [hunk 8]
```diff
@@ -1,336 +1,248 @@
...
-we can install this reference platform as a Crossplane package.
+Now you can install this reference platform. It's packaged as a [Crossplane
+configuration package](https://crossplane.io/docs/v1.9/concepts/packages.html)
+so there is a single command to install this package: 
 
 ```console
-# Check the latest version available in https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/
-kubectl apply -f examples/configuration.yaml
-kubectl get pkg
+up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.4.0
```
Comment (ytsarev) [inline README.md:105]:
```suggestion
up ctp configuration install xpkg.upbound.io/upbound/platform-ref-aws:v0.4.1
```
Let's use the version with the fixed provider-helm dependency

File: README.md [hunk 9]
```diff
@@ -1,336 +1,248 @@
...
+kubectl get providers,providerrevision
 
-## Provision Resources
+kubectl get configurations,configurationrevisions
+```
 
-With the setup complete, we can now use platform-aws to provision resources in AWS.
+Check the
+[marketplace](https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/
+) for the latest version of this platform.
```
Comment (ytsarev) [inline README.md:117]:
```suggestion
Check the
[marketplace](https://marketplace.upbound.io/configurations/upbound/platform-ref-aws/)
for the latest version of this platform.
```

File: README.md [hunk 10]
```diff
@@ -1,336 +1,248 @@
...
 
 ```console
-kubectl apply -f examples/cluster-claim.yaml
-```
+# Create a creds.conf file with the aws cli:
+AWS_PROFILE=default && echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $AWS_PROFILE)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $AWS_PROFILE)" > creds.conf
 
-Verify status:
+# Create a K8s secret with the AWS creds:
+kubectl create secret generic aws-creds -n upbound-system --from-file=creds=./creds.conf
```
Comment (ytsarev) [inline README.md:129]:
```suggestion
kubectl create secret generic aws-creds -n upbound-system --from-file=credentials=./creds.conf
```
fix secret key to match the one from `examples/aws-default-provider.yaml`

File: README.md [hunk 11]
```diff
@@ -1,336 +1,248 @@
...
+kubectl delete -f examples/cluster-claim.yaml,examples/postgres-claim.yaml
 ```
 
-#### Uninstall Provider & Platform Configuration
+To uninstall the provider & platform configuration:
 
 ```console
 kubectl delete configurations.pkg.crossplane.io platform-ref-aws
 kubectl delete providers.pkg.crossplane.io provider-aws
 kubectl delete providers.pkg.crossplane.io provider-helm
```
Comment (ytsarev) [inline README.md:173]:
```suggestion
kubectl delete configurations.pkg.crossplane.io upbound-platform-ref-aws
kubectl delete providers.pkg.crossplane.io upbound-provider-aws
kubectl delete providers.pkg.crossplane.io crossplane-contrib-provider-helm
```
no we need slightly different names to make this command working

File: README.md [hunk 12]
```diff
@@ -1,336 +1,248 @@
...
-```console
-docker login ${REGISTRY} -u ${UPBOUND_ACCOUNT_EMAIL}
-```
+To share your new platform you need to build and distribute this package.
 
-Build package.
+To build the package use the `up xpkg build` command:
 
 ```console
 up xpkg build --name package.xpkg --package-root=package --examples-root="examples"
```
Comment (ytsarev) [inline README.md:217]:
```suggestion
up xpkg build --name package.xpkg --package-root=package --examples-root=examples
```

File: README.md [hunk 13]
```diff
@@ -1,336 +1,248 @@
...
 ```
 
-Push package to registry.
+Afterwards you can it to the marketplace. Don't worry it's private to you.
 
 ```console
-up xpkg push ${PLATFORM_CONFIG} -f package.xpkg
+up repo create ${PLATFORM}
```
Comment (ytsarev) [inline README.md:223]:
```suggestion
TAG=v0.1.0
up repo create ${PLATFORM}
```

File: README.md (hunk 8 above)
Comment (ytsarev) [reply README.md:105]: In addition, I still believe that we should keep `examples/configuration.yaml` and reference it as 'optional GitOps-compatible way of installation'

**Review by ytsarev — CHANGES_REQUESTED** [review]
Please consider suggestions.

File: README.md [hunk 14]
```diff
@@ -1,336 +1,248 @@
...
-Ensure that your kubectl context is pointing to the correct cluster:
+For installing the platform we need a running Crossplane control plane. We are
+using [Universal Crossplane (UXP)
+](https://github.com/upbound/universal-crossplane). Ensure that your kubectl
+context is pointing to the correct Kubernetes cluster or for example create a
+[kind](https://kind.sigs.k8s.io) cluster:
 
 ```console
-kubectl config current-context
+kind create cluster --image kindest/node:v1.25.0
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md:83]: I don't think so.

File: README.md [hunk 15]
```diff
@@ -1,336 +1,248 @@
...
-  name: platform-ref-aws
-spec:
-  package: ${PLATFORM_CONFIG}
-EOF
+open https://marketplace.upbound.io/configurations/${ORG}/${PLATFORM}:v0.0.1
 ```
 
+## Using your custom platform
+
+Now if you want to use it you can follow the steps from above. The only difference is that you need to specify a package-pull-secret as the package is currently private:
```
[the remark this answers is not in the document]
Comment (luebken) [reply README.md:234]: I would say no, as it's done automatically. We could link to some enhanced docs. But I couldn't find a great page currently. I would like to defer this.

[the remark this answers is not in the document]
Comment (ytsarev) [reply README.md:235]: @luebken sorry if I am missing it, at which stage pull secret is getting automatically created in this flow?

Shouldn't
```
up ctp pull-secret create
```
be invoked explicitly before referencing it?

[the remark this answers is not in the document]
Comment (luebken) [reply README.md:235]: Apologies. I thought the command was there. I've added it with https://github.com/upbound/platform-ref-aws/pull/79/commits/91cc513e2e06d9fa0b47d7517c72364c98c28a15.

WDYT?

**Review by ytsarev — COMMENTED** [review]
last nit

**Review by ytsarev — APPROVED** [review]
Awesome, thanks a lot for the mass revamp and all the additional fixes!

**Pull request merged**